doc/man7/EVP_RAND-HMAC-DRBG.pod

```diff
@@ -56,6 +56,23 @@ These parameters work as described in L<EVP_RAND(3)/PARAMETERS>.
 
 =head1 NOTES
 
+When using the FIPS provider, only these digests are permitted (as per
```

Comment:
Same as above.

Reply:
This should be the same as the comment in the above HASH-DRBG doc